core/meta/inc/TClass.h

```diff
@@ -535,7 +543,8 @@ friend class ROOT::Internal::TCheckHashRecursiveRemoveConsistency;
    static void           RemoveClass(TClass *cl);
    static void           RemoveClassDeclId(TDictionary::DeclId_t id);
    static TClass        *GetClass(const char *name, Bool_t load = kTRUE, Bool_t silent = kFALSE);
-   static TClass        *GetClass(const std::type_info &typeinfo, Bool_t load = kTRUE, Bool_t silent = kFALSE);
+   static TClass        *GetClass(const char *name, Bool_t load, Bool_t silent, size_t hint_pair_offset, size_t hint_pair_size);
+   static TClass        *GetClass(const std::type_info &typeinfo, Bool_t load = kTRUE, Bool_t silent = kFALSE, size_t hint_pair_offset = 0, size_t hint_pair_size = 0);
```

Comment:
As this is a public method, would it make sense to swap the last two parameters for backward compatibility?

Or, to break the interface for existing callers and make sure they don't silently get wrong results, to take an `array<size_t, 2>` or a `struct PairInfo_t` as last parameter?

Reply:
The new argument are already additive (require to specify the default arguments).